src/battle-animations.ts

```diff
@@ -666,7 +664,7 @@ class BattleScene {
 		for (let i = 0; i < sidebarIcons.length; i++) {
 			const [iconType, pokeIndex] = sidebarIcons[i];
 			const poke = pokeIndex !== null ? side.pokemon[pokeIndex] : null;
-			const tooltipCode = ` class="picon has-tooltip" data-tooltip="pokemon|${side.n}|${pokeIndex}${iconType === 'pokemon-illusion' ? '|illusion' : ''}"`;
+			const tooltipCode = ` class="picon has-tooltip" data-tooltip="pokemon|${this.battle.sides.indexOf(side)}|${pokeIndex}${iconType === 'pokemon-illusion' ? '|illusion' : ''}"`;
```

Comment:
```suggestion
			const tooltipCode = ` class="picon has-tooltip" data-tooltip="pokemon|${side.n}|${pokeIndex}${iconType === 'pokemon-illusion' ? '|illusion' : ''}"`;
```
This shouldn't be necessary, right? This should just be `side.n`.